test/TestHelper.cpp

```diff
@@ -172,6 +173,61 @@ void ImportTest::importTransaction(json_spirit::mObject& _o)
 	}
 }
 
+bool ImportTest::compareStates(State const& _stateExpect, State const& _statePost)
+{
+	//TestNet Addresses
+	static Addresses testNetAddressList = boost::assign::list_of
```

Comment:
you don't need that. You can use this constructor to get an empty state:
myState(Address(), OverlayDB(), eth::BaseState::Empty)